gradle/libs.versions.toml

```diff
@@ -52,8 +52,8 @@ jetbrains-grammarkit = "org.jetbrains.grammarkit:2021.2.2"
 jetbrains-intellij = "org.jetbrains.intellij:1.7.0"
 # @pin because as per https://plugins.jetbrains.com/docs/intellij/kotlin.html#kotlin-standard-library:
 #      "If a plugin supports multiple platform versions, it must (...) target the lowest bundled stdlib version"
-#      which is 1.4.0 in our case (as this is the Kotlin version bundled into IntelliJ 2020.3).
-jetbrains-kotlin = "org.jetbrains.kotlin.jvm:1.4.0"
+#      which is different in our case (as this is the Kotlin version bundled into IntelliJ 2021.2).
+jetbrains-kotlin = "org.jetbrains.kotlin.jvm:1.5.10"
```

Comment:
With this change, you can probably remove the `@pin` on `io.freefair.aspectj.post-compile-weaving`... pls check if the project still builds once that plugin is bumped to `6.5.0`

Reply:
i ve tried that. it does not work yet with kotlin jvm 1.5.10. I managed to get
![image](https://user-images.githubusercontent.com/19799111/182881426-d6455543-6674-4280-b1f0-545303f068a0.png)
and the already known https://github.com/freefair/gradle-plugins/issues/554 too

Reply:
Oh! okay, so only works with Kotlin 1.6? did you already update on another PR?

Reply:
sadly it's neither working with 1.6 :/ I am getting the same errors there as here

Reply:
Huh could you open a separate issue for that? 🤔

Reply:
Maybe some non-trivial migration will be necessary. Plus update the comment on `@pin` accordingly

Reply:
I ve created an issue (https://github.com/freefair/gradle-plugins/issues/578).

Reply:
I spet some time on this one but I see no solution for now. Comment left and let's see what they say.

Reply:
Whoops okay, so **not** a matter of Kotlin incompatibility this time ☹️